openshift_tools/monitoring/awsutil.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+'''
+    Interface to awscli
+'''
+# vim: expandtab:tabstop=4:shiftwidth=4
+#
+#   Copyright 2015 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# Disabling invalid-name because pylint doesn't like the naming conention we have.
+# pylint: disable=invalid-name
+
+import json
+import os
+import shlex
+import subprocess
+
+class AWSUtil(object):
+    ''' Routines for interacting with awscli '''
+
+    def __init__(self, access_key, secret_access_key, verbose=False):
+        ''' Save auth details for later usage '''
+
+        self.access_key = access_key
+        self.secret_access_key = secret_access_key
```

Comment:
I think we should default to using the environment variables if the client of this library didn't pass in these values.

That seems to be the pattern that libraries that work with AWS use.